db/set.yaml

```diff
@@ -0,0 +1,10 @@
+name: string (Get started with Data Science)
+id: unique_string (s10101)
+description: string (New to Data Science? Attend these live classes to improve your skill!)
+showMoreState: boolean (yes/no)
```

Comment:
I am not sure if this field will see much use, if any. I thought all of our sets would have the `showMore` off by default and have the `Show More` button on there. @geooot What do you think of this field?

Reply:
i believe George mentioned having an option in the issue.

Reply:
@tatiaris Yes, I did want this feature (basically so we could say the first set could be fully shown, etc)

Reply:
@geooot @tatiaris  My concern with that is with the fact that we have priorities for sets too, so if the priorities change around and the ordering of sets changes, and we forget to change this value, then the wrong sets might be fully shown. If we want the first set to be fully shown, we could have the `Page` component specify that.

However, if we think we'll want other sets on the page to be shown and the problem I mentioned above would be rare, then we can close this comment

Reply:
left it there for now :thinking: